Hatman/Commands/Dice.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
+using System;
```

Comment:
I don't know what happened here. A funny character (BOM?) appeared in my browser and I deleted it.